### PR TITLE
Correctly passing caps when getting screen orientation

### PR DESCRIPTION
**Pull request by vania-pooh**

### CODE REVIEW

**Comment by vania-pooh** [comment]
Fixed.

File: lib/commands/actions.js [hunk 1]
```diff
@@ -29,7 +29,10 @@ commands.longPressKeyCode = async function (keycode, metastate = null) {
 };
 
 commands.getOrientation = async function () {
-  let orientation = await this.bootstrap.sendAction("orientation", {});
+    let params = {
```
Comment (imurchie) [inline lib/commands/actions.js]: This indentation is two spaces too far. Can you run eslint locally (`gulp eslint`)?

**Comment by vania-pooh** [comment]
Done. Finally understood that IntellijIDEA just skips pre-commit checks. Only works when doing commits in command line.

**Comment by imurchie** [comment]
Interesting. Ok. We'll let CI run but this all looks good. Thanks!

**Comment by vania-pooh** [comment]
@imurchie any updates?

**Comment by imurchie** [comment]
Sorry. I got sidetracked trying to fix the CI. This seems good.

**Comment by imurchie** [comment]
Published in `appium-android-driver@1.23.0`.

**Pull request merged**